### DISCUSSION TITLE
OpenGoPo <=> GoPro Labs?

### DISCUSSION THREAD

**Discussion by dimitribeyondgravity** [gopro/OpenGoPro#606] [Q&A] [answered]
Hi !
OpenGoPro and GoPro Labs are incredibly useful !
I'm just wondering why it's not possible (from what i know) to perform some actions with OpenGoPro, that we already have with GoPro Labs.

For exemple:
With GoPro labs: "some text", will display a message on the screen.
With OpenGoPro, it doesn't seem to be possible to do so.
Why not?

Same goes for tons of actions:
- lock the screen
- use the archive mode
- metadata injection (4C)
- etc
We are trying to perform actions like those ones, through OpenGoPro, and it's very frustrating to see it's working with a qr code, but there no command with the API 😓

Thanks for any help or idea to close that gap !

**Comment by tony-gutierrez** [comment]
Same for reboot...labs can, open gopro cant

**Comment by kmatzen** [comment]
Seems like two completely separate teams.

**Comment by tcamise-gpsw** [comment, marked as answer]
Yeah Open GoPro is just a documentation (and demos, etc) of the API's that the official firmware uses.

GoPro Labs is custom firmware to load on the camera.
